perfkitbenchmarker/pkb.py

```diff
@@ -331,23 +332,17 @@ def _LogCommandLineFlags():
   logging.info('Flag values:\n%s', '\n'.join(result))
 
 
-def RunBenchmarks(publish=True):
-  """Runs all benchmarks in PerfKitBenchmarker.
-
-  Args:
-    publish: A boolean indicating whether results should be published.
+def SetUpPKB():
+  """Set globals and environment variables for PKB.
 
-  Returns:
-    Exit status for the process.
+  Having this function separate from functions that do any work lets
+  us call this to set up integration tests.
```

Comment:
I don't think this comment is necessary in the code itself, but it might be nice to elaborate here on what this function does (for example, I see it creates a temp dir on the local OS, which isn't mentioned yet)